ui/src/router/components/form/components/router_config/RulesPanel.js

```diff
@@ -50,17 +77,44 @@ export const RulesPanel = ({ rules, routes, onChangeHandler, errors = {} }) => {
         </Fragment>
       }>
       <EuiFlexGroup direction="column" gutterSize="s">
+        {
+          rules.length > 0 ?
```

Comment:
Can re-write this as below more concisely, instead of the ternary operator:
```javascript
    {
          rules.length > 0 && (<EuiFlexItem key="default-route"> ... </EuiFlexItem>)
    }
```